sharktank/sharktank/types/tensors.py

```diff
@@ -538,15 +538,38 @@ def unsqueeze(self, dim: int) -> "AnyTensor":
 
         return unsqueeze(self, dim)
 
+    @overload
+    def view(self, dtype: torch.dtype) -> "AnyTensor":
+        ...
+
+    @overload
     def view(self, *args: Union[List[List[int]], List[int]]) -> "AnyTensor":
+        ...
+
+    def view(
+        self,
+        *args: Union[List[List[int]], List[int], torch.dtype],
+        dtype: torch.dtype | None = None,
```

Comment:
We have Tensor.to that needs to handle a similar case, but that one is implemented in a different way, the main func takes (*args, **kwargs). I think it would be better to stay consistent with how we implement these overloaded funcs.

Reply:
I don't think 
```
def view(self, *args, **kwargs)
```
is more understandable compared to
```
def view(
    self,
    *args: Union[List[List[int]], List[int], torch.dtype],
    dtype: torch.dtype | None = None,
)
```

Why not make every function accept just `*args, **kwargs` then?